tilewe/src/ctilewemodule.c

```diff
@@ -225,6 +225,89 @@ static PyObject* Board_NumPlayerPcs(BoardObject* self, PyObject* args, PyObject*
     return NULL; 
 }
 
+static PyObject* Board_PlayerOpenCorners(BoardObject* self, PyObject* args, PyObject* kwds) 
```

Comment:
`board.player_corners` in Python; a list of `tilewe.Tile` (i.e. integers 0 to 399) representing where the requested player has open corners